### PR TITLE
Feature/next steps

### PR DESCRIPTION
**Pull request by AparnaPattathil**

### CODE REVIEW

**Review by dominikgold — APPROVED** [review]

File: home-connect/build.gradle
Comment (falkorichter) [inline home-connect/build.gradle]: this should not be on develop - it is only for the submodule based integration that was setup for quicker iteration

**Review by falkorichter — CHANGES_REQUESTED** [review]
let´s wait until the feature is completely done and merge it then

File: .idea/codeStyles/Project.xml
Comment (falkorichter) [inline .idea/codeStyles/Project.xml:3]: we should ignore the .idea folder and remove the file from version control

**Pull request merged**
